components/ws-manager-mk2/controllers/status.go

```diff
@@ -59,6 +59,10 @@ func (r *WorkspaceReconciler) updateWorkspaceStatus(ctx context.Context, workspa
 	defer func() {
 		if oldPhase != workspace.Status.Phase {
 			log.Info("workspace phase updated", "oldPhase", oldPhase, "phase", workspace.Status.Phase)
+			if workspace.Status.Phase == workspacev1.WorkspacePhaseStopping {
```

Comment:
Ah, nice place to do this!

Should we extend the check to this, so we don't overwrite again?
```suggestion
			if workspace.Status.Phase == workspacev1.WorkspacePhaseStopping && workspace.Status.PodStoppingTime == nil {
```

Reply:
Once the deploy is done, I will test again + approve, then we can :heavy_check_mark: and merge. :tada:

Reply:
> There is a if for old status in above

Ah true, so it's probably even best this way! :+1: 

> Considering that the pod may be recreated

We don't have to consider this, as Status is "completely" cleared on re-creation. :+1: